src/main/java/io/polivakha/mojo/properties/AbstractWritePropertiesMojo.java

```diff
@@ -24,118 +24,127 @@
 import org.apache.maven.plugins.annotations.Parameter;
 import org.apache.maven.project.MavenProject;
 
+import java.io.BufferedReader;
 import java.io.File;
 import java.io.FileNotFoundException;
-import java.io.FileOutputStream;
 import java.io.IOException;
-import java.io.OutputStream;
 import java.io.PrintWriter;
 import java.io.StringReader;
 import java.io.StringWriter;
+import java.nio.charset.StandardCharsets;
 import java.util.ArrayList;
-import java.util.Collections;
+import java.util.Comparator;
 import java.util.List;
 import java.util.Properties;
-import java.io.BufferedReader;
 
 /**
  * @author <a href="mailto:zarars@gmail.com">Zarar Siddiqi</a>
+ * @author Sergey Korotaev
  */
 public abstract class AbstractWritePropertiesMojo
-    extends AbstractMojo
-{
+        extends AbstractMojo {
 
-    @Parameter( defaultValue = "${project}", required = true, readonly = true )
+    @Parameter(defaultValue = "${project}", required = true, readonly = true)
     private MavenProject project;
 
-    @Parameter( required = true, property = "properties.outputFile" )
+    @Parameter(required = true, property = "properties.outputFile")
     private File outputFile;
 
+    @Parameter(property = "sort", defaultValue = "true")
```

Comment:
default value as `true` here I guess is not right, let's stick to `false` here. Leave this choice to the user.

Reply:
true/false means that properties form file will be sort asc/desc accordingly, because of true id default for asc sorting

Reply:
I understood the logic, that's not what I am asking. Regarding my previous comment on sort parameter, please, do not set anything here for user